### PR TITLE
Fixed formatting.

### PR DESCRIPTION
**Pull request by bryanjnelson**

### CODE REVIEW

File: chapter6/index.md [hunk 1]
```diff
@@ -197,7 +197,7 @@ The output is now as follows:
 
 ```
 {default::NPC {name: 'Jonathan Harker', lover: default::NPC {name: 'Mina Murray'}}}```
```
Comment (raddevon) [inline chapter6/index.md:199]:
```suggestion
{default::NPC {name: 'Jonathan Harker', lover: default::NPC {name: 'Mina Murray'}}}
```

File: chapter6/index.md [hunk 2]
```diff
@@ -197,7 +197,7 @@ The output is now as follows:
...
-
+```
 Success!
```
Comment (raddevon) [inline chapter6/index.md:201]: We try to leave a line between a code block and other text.
```suggestion

Success!
```

**Review by raddevon — CHANGES_REQUESTED** [review]
Thank you for this! I have two minor suggestions before we merge this.

File: chapter6/index.md (hunk 1 above)
Comment (bryanjnelson) [reply chapter6/index.md:199]: Missed that!

**Review by raddevon — APPROVED** [review]

**Pull request merged**